db/static_seeds.rb

```diff
@@ -8,3 +8,6 @@
   {start_date: "2017-05-28", name: "Bio", video: nil, duration: nil},
   {start_date: "2017-05-28", name: "Konventionell", video: nil, duration: nil}
 ])
+Sensor.create!([
```

Comment:
Does this really have to go into the `static_seeds`? Only include those records, that you want to have in all tests.